packages/shell-api/src/shell-bson.ts

```diff
@@ -191,7 +192,7 @@ export default function constructShellBson(bson: typeof BSON, printWarning: (msg
       assertArgsDefinedType([hexstr], ['string'], 'UUID');
       // Strip any dashes, as they occur in the standard UUID formatting
       // (e.g. 01234567-89ab-cdef-0123-456789abcdef).
-      const buffer = Buffer.from((hexstr as string).replace(/-/g, ''), 'hex');
+      const buffer = Buffer.from((hexstr as any).replace(/-/g, ''), 'hex');
```

Comment:
This looks a bit weird, string should be a good enough assertion here 🤔

Reply:
Hmm. Let me check again. I have zero recollection of this :)

Reply:
Oh I think it was just an early case of the linting complaining that the assertion was unnecessary even though if you remove it it complains that it so is. Unnecessary since we disable that rule anyway. Will revert it.